configuration/configuration.go

```diff
@@ -174,6 +174,9 @@ type Configuration struct {
 		// Addr specifies the the redis instance available to the application.
 		Addr string `yaml:"addr,omitempty"`
 
+		// Usernames can be used as a finer-grained permission control since the introduction of the redis 6.0.
+		Username string `yaml:"username,omitempty"`
```

Comment:
What happens if you can't use a lower version than v6?

Reply:
The username is an optional item so that user don't need to specify the username if the user uses lower version, they only need to input the username if user new version and enable ACL in redis.